params/config_arbitrum.go

```diff
@@ -57,6 +57,7 @@ type ArbitrumChainParams struct {
 	GenesisBlockNum           uint64
 	MaxCodeSize               uint64 `json:"MaxCodeSize,omitempty"`     // Maximum bytecode to permit for a contract. 0 value implies params.DefaultMaxCodeSize
 	MaxInitCodeSize           uint64 `json:"MaxInitCodeSize,omitempty"` // Maximum initcode to permit in a creation transaction and create instructions. 0 value implies params.DefaultMaxInitCodeSize
+	MaxUncompressedBatchSize  uint64 `json:"MaxUncompressedBatchSize,omitempty"`
```

Comment:
Chains that don't have `MaxUncompressedBatchSize` set already will initialize it to zero which causes a backwards incompatibility issue. I'd suggest accessing `MaxUncompressedBatchSize` through a `ChainConfig` accessor method like we have below and setting a default there to the original value if `MaxUncompressedBatchSize == 0`. I think it means you'll need to pass around a `ChainConfig` instead of an `ArbitrumChainParams` object.